### ISSUE TITLE
Swagger documentation shows wrong Example Value if multiple Types are used

### ISSUE THREAD

**Issue by mnisius** [stoicflame/enunciate#1129]
Hi i noticed that the latest enunciate version shows a wrong Example Value if an Endpoint has multiple return types. 

My Code looks like this
```java
@StatusCodes({
  @ResponseCode(code = 500, condition = "error", type = @TypeHint(Error.class))
  //Same result with :  @ResponseCode(code = 200, condition = "success", type = @TypeHint(Result.class))
})
@GetMapping("test")
public Result test()  {
	return null;
}
```

So I would except Code 200 shows Example Value of Type Result.class und Code 550 shows Example Values of Type Error.class.
But the example Value always shows Type Result.class

![wrong](https://user-images.githubusercontent.com/56688244/185368525-9d2d7462-4013-4c42-ba5a-7eef19f42994.png)

The Model however is correct

![correct](https://user-images.githubusercontent.com/56688244/185368573-d6afd5a8-e8d0-4598-8656-5d7a72f23084.png)

**Comment by stoicflame** [comment]
Thank you for the report.

**Comment by stoicflame** [comment]
This is fixed in version 2.15.0.
